src/hooks/screen/useCreateChat.ts

```diff
@@ -124,6 +124,22 @@ const useCreateChat = () => {
     }
   };
 
+  const createAnonymousChat = async (selectedUser) => {
+    try {
+      setLoadingCreateChat(true);
+      const response = await getOrCreateAnonymousChannel(selectedUser?.user?.userId);
+      const chatData = createChannelJson(response, selectedUser);
+      const channelList = ChannelList.fromMessageAnonymouslyAPI(chatData);
+      await channelList.saveIfLatest(localDb);
+      handleMemberSchema(response);
+      goToChatScreen(channelList, GROUP_INFO);
+    } catch (e) {
+      console.log({e}, 'error create chat');
+    } finally {
+      setLoadingCreateChat(false);
+    }
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `createAnonymousChat` function is missing error handling. Currently, if an error occurs, it's only logged to the console and not handled in any other way. This could lead to unexpected behavior or crashes in your application. Consider adding more robust error handling, such as displaying a user-friendly error message or retrying the operation.

```diff
  const createAnonymousChat = async (selectedUser) => {
    try {
      setLoadingCreateChat(true);
      const response = await getOrCreateAnonymousChannel(selectedUser?.user?.userId);
      const chatData = createChannelJson(response, selectedUser);
      const channelList = ChannelList.fromMessageAnonymouslyAPI(chatData);
      await channelList.saveIfLatest(localDb);
      handleMemberSchema(response);
      goToChatScreen(channelList, GROUP_INFO);
    } catch (e) {
-        console.log({e}, 'error create chat');
+        // Display a user-friendly error message
+        SimpleToast.show('Error creating anonymous chat. Please try again.');
+        // Optionally, you can rethrow the error to be caught by a higher-level error handler
+        throw e;
    } finally {
      setLoadingCreateChat(false);
    }
  };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->